CreeDictionary/CreeDictionary/templates/CreeDictionary/components/paradigm.html

```diff
@@ -39,7 +39,19 @@
                       <td class="paradigm-cell paradigm-cell--empty"></td>
                     {% else %}
                       {# TODO: split this over multiple rows #}
-                      <td class="paradigm-cell">{% orth cell %}</td>
+
+                      {# observed form #}
+                      {% if cell.frequency != 0 %}
```

Comment:
```suggestion
                      {% if cell.frequency > 0 %}
```

For some reason, that is easier for me to understand.